### DISCUSSION TITLE
Can you merge the scene information from 2 xml files?

### DISCUSSION THREAD

**Discussion by Disgruntoad** [mitsuba-renderer/mitsuba3#1403] [Q&A] [answered]
Hi all, 

I'm curious if there is a way to merge the scene information from 2 xml files (for instance, one containing objects and a second containing emitters). I was reading through the api and 'editing a scene' documentation pages, but it wasn't obvious to me how I would actually do this.

Thanks for any help!

**Comment by merlinND** [comment, marked as answer]
Hello @Disgruntoad,

Changing a scene's contents (i.e. plugins, shapes, etc) after it has been loaded is not supported, however you can:

1. Parse the XML files and combine them on-the-fly in Python
2. Or, see this discussion for how to create a new scene from an existing Scene: https://github.com/mitsuba-renderer/mitsuba3/issues/1328
